app/assets/javascripts/Map.elm

```diff
@@ -83,4 +88,7 @@ encodeFacility facility =
     object
         [ ( "id", int facility.id )
         , ( "position", encodeLatLng facility.position )
+        , ( "facilityType", string facility.facilityType )
+          -- TODO rename field
```

Comment:
I would name it facilityPriority and that's it. Anything else regarding the name?